lahja/base.py

```diff
@@ -184,10 +187,10 @@ def broadcast_nowait(
         ...
 
     @abstractmethod
-    def subscribe(
+    async def subscribe(
```

Comment:
One thing I just noticed: We *might* need a `subscribe_nowait` API now because the `subscribe` API could previously used from synchronous methods. Well there's a good chance that we might be able to convert these places into `async` methods but some could be tricky.

Reply:
I also realized that I'm using the wrong `inspect` function and that asynchronous subscription handlers don't appear to be working in some cases.  I have a fix in one of my branches that I'll open up shortly.